worldedit-core/src/main/java/com/sk89q/worldedit/math/MathUtils.java

```diff
@@ -105,4 +105,20 @@ public static double dSin(double degrees) {
         return Math.sin(Math.toRadians(degrees));
     }
 
+    /**
+     * Returns the rounded double of the given value. This is needed because
+     * {@code Math.round(value)} implementation rounds a.5, where a is an integer,
+     * regardless of sign, to the closest greater integer.
+     * ("Round up")
+     *
+     * In other words, {@code Math.round(-3.5)}, as an example, rounds -3.5 to -3
+     * instead of the "typical" -4. The following function has the "typical"
+     * behavior for both positive and negative values.
+     *
+     * @param value the value
+     * @return the rounded value
+     */
+    public static double round(double value) {
```

Comment:
It doesn't round to the nearest even number. It rounds to the nearest integer. The difference is the behavior when the fractional part is exactly 0.5 or 1/2. But we'll change the description to better reflect that. We just meant "typical" as in the way it's taught in basic arithmetic, despite the ambiguity in computing contexts.